src/test/java/krona/KronaTest.java

```diff
@@ -0,0 +1,4 @@
+package krona;
+
+public class KronaTest {
```

Comment:
I think having a JUnit test for the Krono app itself might be a bit impractical? Since you would have to isolate many of its dependent classes with SUTs and test drivers, but definitely not impossible